engine/gui/src/gui.cpp

```diff
@@ -267,6 +267,11 @@ namespace dmGui
 
     void SetPhysicalResolution(HContext context, uint32_t width, uint32_t height)
     {
+        // If window size changed at the initialization phase, gui context doesn't exist.
```

Comment:
Not sure this should be here.
What code called this function without a context?

If it's from engine.cpp OnWindowSize() then I think the null ptr check should be there since
1) we don't do defensive programming
2) the rest of the guicpp doesn't check the context pointer

Reply:
Yes, I didn't like that either. Thank you!